docs/svelte-testing-library/setup.mdx

```diff
@@ -4,85 +4,90 @@ title: Setup
-with any testing framework and runner you're comfortable with.
+We recommend using [Vitest](https://vitest.dev/) but you're free to use the
+library with any testing framework and runner you're comfortable with.
 
 ## Vitest
-1.  Install Vitest and jsdom
 
-    ```
-    npm install --save-dev vitest jsdom
-    ```
+1. Install Vitest and jsdom
 
-2.  Add the following to your `package.json`
+We're using `jdom` here as the test environment, but you can use any other
+options e.g `happy-dom`.
 
-    ```json
-    {
-      "scripts": {
-        "test": "vitest run src",
-        "test:watch": "vitest src"
-      }
-    }
-    ```
+```bash npm2yarn
+npm install --save-dev vitest jsdom
+```
 
-3.  You'll need to compile the Svelte components before using them in Vitest, so
-    you need to install
-    [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte) and Vite
-    
+Optionally install `@vitest/ui` open a UI within a browser window to view and
+interact with your tests.
 
-    ```
-    npm install --save-dev @sveltejs/vite-plugin-svelte vite
-    ```
+```bash npm2yarn
+npm install --save-dev @vitest/ui
+```
 
-4.  Add a `vitest.config.ts` configuration file to the root of your project
+1. Add the test scipts to your `package.json` to run the tests with Vitest
 
-    ```js
-    import { defineConfig } from 'vitest/config';
-    import { svelte } from '@sveltejs/vite-plugin-svelte';
+   ```json
+   {
+     "scripts": {
+       "test": "vitest run",
+       "test:ui": "vitest --ui",
+       "test:watch": "vitest"
+     }
+   }
+   ```
 
-    export default defineConfig({
-        plugins: [svelte({ hot: !process.env.VITEST })],
-        test: {
-            include: ['src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-            globals: true,
-            environment: 'jsdom'
-        }
-    });
+2. To compile the Svelte components before using them in Vitest, you need to
+   install
+   [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte)
+   and Vite
 
-    ```
+   ```bash npm2yarn
+   npm install --save-dev @sveltejs/vite-plugin-svelte vite
+   ```
 
-5.  This is optional but it is recommended, you can install
-    [jest-dom](https://github.com/testing-library/jest-dom) to add handy
-    assertions to Jest
+3. Add a `vitest.config.ts` configuration file to the root of your project
 
-    5.1 Install `jest-dom`
+   ```js
+   import {defineConfig} from 'vite'
+   import {svelte} from '@sveltejs/vite-plugin-svelte'
 
-    ```
-    npm install --save-dev @testing-library/jest-dom
-    ```
+   export default defineConfig({
+     plugins: [svelte({hot: !process.env.VITEST})],
+     test: {
+       globals: true,
+       environment: 'jsdom',
+     },
+   })
+   ```
 
-    5.2 import `@testing-library/jest-dom` at the start of your test files
+4. Optionally install [jest-dom](https://github.com/testing-library/jest-dom) to
```

Comment:
I'm hesitating about this phrasing since the go-to solution for `vitest` is probably `vitest-dom` and not `jest-dom`... Wdyt?

Reply:
Yea, I had the same thought and that's when I reached out to you earlier.
I didn't update it because it was originally `jest-dom`, and it isn't maintained by us.